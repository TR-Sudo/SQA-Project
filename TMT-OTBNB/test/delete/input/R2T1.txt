Login
AdminUser
Delete
username@20
username05
Logout
quit