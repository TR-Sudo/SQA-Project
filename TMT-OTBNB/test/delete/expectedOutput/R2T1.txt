Enter transaction: Enter username: You have been logged in
Enter transaction: Enter username: Invalid username: username must be less than 15 characters and can only contain A-z, numbers, - and _
Enter username: Deleted user username05
Enter transaction: You have been logged out
Enter transaction: 